gwpy/types/tests/test_series.py

```diff
@@ -182,9 +182,14 @@ def test_getitem_index(self, array):
         assert len(newarray) == len(newarray.value)
         assert len(newarray.value) == len(newarray.xindex)
 
+    def test_getitem_list_index(self, array):
+        """Test that __getitem__ works with list and numpy.array"""
+        indices = numpy.array([0, 1, len(array) - 1])
+        lindices = [0, 1, len(array) - 1]
+        assert numpy.all(array[indices].value == array[lindices].value)
```

Comment:
Here you can use `assert_quantity_sub_equal` to directly compare the two array objects
```suggestion
        assert utils.assert_quantity_sub_equal(array[indices].value, array[lindices].value)
```